.github/workflows/tf_plan_production.yml

```diff
@@ -26,8 +26,9 @@ jobs:
       matrix:
         include:
           - module: hosted_zone
-          - module: api
+          - module: network
```

Comment:
Appreciate you changing the matrix order here, but because we have no [`max-parallel` setting](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#defining-the-maximum-number-of-concurrent-jobs) on the matrix itself, all these module TF plan commands will happen at the same time.

It should be fine though, and if not, we can adjust down the road.